README.md

```diff
@@ -1 +1,252 @@
 # GWT Rules for Bazel
+
+<div class="toc">
+  <h2>Rules</h2>
+  <ul>
+    <li><a href="#gwt_application">gwt_application</a></li>
+  </ul>
+</div>
+
+## Overview
+
+These build rules are used for building [GWT](http://www.gwtproject.org/)
+applications with Bazel. Applications are compiled as `.war` files containing
+compiled JavaScript and other resources. GWT applications can also be run in
+[Development Mode](http://www.gwtproject.org/doc/latest/DevGuideCompilingAndDebugging.html)
+via `bazel run`.
+
+<a name="setup"></a>
+## Setup
+
+To be able to use the GWT rules, you must provide bindings for the following
+targets:
+
+  * `//external:asm`
+  * `//external:javax-validation`
+  * `//external:java-validation-src`
+  * `//external:gwt-dev`
+  * `//external:gwt-user`
+
+The easiest way to do so is to add the following to your `WORKSPACE` file:
+
+```python
+http_archive(
+  name = "io_bazel_rules_gwt",
+  url = "https://github.com/bazelbuild/rules_gwt/archive/0.0.1.tar.gz",
+  sha256 = "TODO",
```

Comment:
Removed sha256. @jart suggested `http_archive` over `git_repository` in https://github.com/bazelbuild/rules_groovy/pull/4; I have no strong feelings either way.